From d3de1750f90db41ae89964b9bce156a77995dbcd Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 28 Nov 2017 22:12:21 +0800
Subject: [PATCH 2929/5242] MLK-16604-2 arm64: dts: imx8mq: add
 usb3-resume-missing-cas for usb3

commit  17b69e8deccc5c43f5024d0a00a7608d57301632 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add usb3-resume-missing-cas property for imx8mq usb3 to work around
the usb3 resume if the usb3 device plugged in while in system sleep.

Signed-off-by: Li Jun <jun.li@nxp.com>
Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index e695347..026d363 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -887,6 +887,7 @@
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&usb_otg1_pd>;
 			snps,power-down-scale = <2>;
+			usb3-resume-missing-cas;
 			status = "disabled";
 		};
 	};
@@ -926,6 +927,7 @@
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&usb_otg2_pd>;
 			snps,power-down-scale = <2>;
+			usb3-resume-missing-cas;
 			status = "disabled";
 		};
 	};
-- 
1.7.9.5

